src/addons/dispatch.js

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import Dispatcher from '../Dispatcher';
 import getDisplayName from './getDisplayName';
 
-export default function dispatch(stores) {
-  return DecoratedComponent => class ReduxDispatcherDecorator {
-    static displayName = `ReduxDispatcher(${getDisplayName(DecoratedComponent)})`;
+export default function dispatch(store) {
+  return DecoratedComponent => class DispatcherDecorator {
+    static displayName = `Dispatcher(${getDisplayName(DecoratedComponent)})`;
 
     render() {
       return (
-        <Dispatcher stores={stores}>
+        <Dispatcher store={store}>
```

Comment:
The whole act of hot reloading any Store now is just replacing this single prop.